index.html

```diff
@@ -142,6 +142,18 @@ <h5>Languages</h5><img src="./images/Union.png" alt="expand" class="arrow"></li>
           <li class="expertise"><h5>Skills</h5><img src="./images/Union2.png" alt="expand" class="arrow"></li>
         </ul>
       </section>
+
+      <section id="form-section">
+        <h2>contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built ora project that needs coding. I’d love to help with it </p>
+
+        <form id="form" action="https://formspree.io/f/xjvlpwvy" method="post">
+          <input id="name" name="name" type="text" maxlength="30" placeholder="Yeremias" required>
+          <input id="email" name="email" type="email" placeholder="natajayanj14@gmail.com" required>
+          <textarea id="text" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+          <button type="submit">Get It Touch</button>
```

Comment:
- [ ] [OPTIONAL] it would be better if you implement the button effect on hover and press as in the Figma Design 
![image](https://user-images.githubusercontent.com/61790587/165638422-5d857de0-72a2-49af-b844-1d294cda83fb.png)